cmake/modules/FindSpdlog.cmake

```diff
@@ -0,0 +1,42 @@
+# FindSpdlog
+# -------
+# Finds the Spdlog library
+#
+# This will define the following variables::
+#
+# SPDLOG_FOUND - system has Spdlog
+# SPDLOG_INCLUDE_DIRS - the Spdlog include directory
+# SPDLOG_LIBRARIES - the Spdlog libraries
+# SPDLOG_DEFINITIONS - the Spdlog compile definitions
+#
+# and the following imported targets::
+#
+#   Spdlog::Spdlog   - The Spdlog library
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_SPDLOG spdlog QUIET)
```

Comment:
since older versions floating around on linux dists don't seem to work, we have to specify a minimum version here